src/main/java/it/gov/pagopa/pu/sil/service/ingestionflowfile/IngestionFlowFileAuthorizationService.java

```diff
@@ -82,7 +82,7 @@ private IngestionFlowFileRequestDTO mapToReservationRequest(IngestionFlowFileTyp
       .filePathName(UNKNOWN)
       .fileName(UNKNOWN)
       .fileSize(0L)
-      .ingestionFlowFileType(ingestionFlowFileType)
+      .ingestionFlowFileType(IngestionFlowFileRequestDTO.IngestionFlowFileTypeEnum.valueOf(ingestionFlowFileType.name()))
```

Comment:
let's not use name, but toString or getValue
```suggestion
      .ingestionFlowFileType(IngestionFlowFileRequestDTO.IngestionFlowFileTypeEnum.valueOf(ingestionFlowFileType.toString()))
```